biogrid/libs/biogrid-simulator/src/lib/bioenergy-source/solar-panel.ts

```diff
@@ -30,6 +34,8 @@ export class SolarPanel extends EnergySource {
       );
     }
     this.sizeSqMtr = sizeSqMtr;
+    this.name = name;
```

Comment:
is this supposed to be a unique name? looks like it defaults to GRID_ITEM_NAMES.SOLAR_PANEL, what happens if multiple panels have the same name? is that okay?

Reply:
It is unique, I implemented it with a default for cases where there is just one solar panel. One doesn't have to pass the name, but I will make it compulsory since it will be unique at the end of the day.